<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-UFMB1VQ" failures="6" tests="93" name="Default test" time="1695.566" errors="0" timestamp="2023-01-20T19:04:44 EST">
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_1" time="1.696"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_10" time="1.428"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_11" time="1.208"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_12" time="37.984"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_13" time="26.859"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_14" time="1.887"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_15" time="1.891"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_16" time="1.179"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_17" time="0.305"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_18" time="1.185"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_19" time="0.351"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_2" time="0.292"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_20" time="1.083"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_21" time="7.06"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_22" time="11.931"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_23" time="13.971"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_24" time="13.234"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_25" time="22.207"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_26" time="6.38"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_27" time="11.41"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_28" time="13.332"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_29" time="30.551"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_3" time="1.048"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_30" time="12.93"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_31" time="1.072"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_31_1" time="20.859"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_32" time="24.155"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_33" time="33.987"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_34" time="20.659"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_35" time="48.052"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_36" time="27.98"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_37" time="23.596"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_38" time="26.095"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_39" time="0.986"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_4" time="5.709"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_40" time="7.006"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_41" time="8.057"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_42" time="1.065"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_43" time="1.041"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_44" time="0.527"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_45" time="7.242"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_46" time="0.672">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.ProductListingPage.setDocumentTileLine(ProductListingPage.java:352)
at com.pwc.productcentral.SanityTestCases.WEB_46(SanityTestCases.java:550)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_46 -->
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_47" time="0.84"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_48" time="11.844"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_49" time="1.482"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_5" time="4.882"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_50" time="14.967"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_51" time="23.049"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52" time="1.095">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.BasePage.setAllImages(BasePage.java:320)
at com.pwc.productcentral.SanityTestCases.WEB_52(SanityTestCases.java:606)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_52 -->
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_1" time="0.488"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_2" time="1.109"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_3" time="0.561"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_4" time="1.181"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_5" time="22.439"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_53" time="21.403"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_54" time="25.228"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_55" time="21.03"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_56" time="23.621"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_57" time="27.498"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_58" time="21.848">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.ProductPage.setFirstTabActive(ProductPage.java:641)
at com.pwc.productcentral.SanityTestCases.WEB_58(SanityTestCases.java:718)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_58 -->
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_59" time="22.123"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_5_2" time="9.534"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_6" time="1.056"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_60" time="21.667"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_61" time="45.685"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_62" time="34.228"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_63" time="49.141"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_64" time="29.77"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_65" time="44.662">
    <failure type="java.lang.AssertionError" message="expected [Last updated 01/09/2023] but found [Last updated 01/20/2023]">
      <![CDATA[java.lang.AssertionError: expected [Last updated 01/09/2023] but found [Last updated 01/20/2023]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Pages.AuthoringPage.setUpdatedDateFromAssets(AuthoringPage.java:1415)
at com.pwc.productcentral.SanityTestCases.WEB_65(SanityTestCases.java:784)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_65 -->
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_66" time="16.848">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.AuthoringPage.setUpdatedDateVisibility(AuthoringPage.java:1469)
at com.pwc.productcentral.SanityTestCases.WEB_66(SanityTestCases.java:793)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_66 -->
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_67" time="31.447"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_68" time="15.882">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.HomePage.setSearchResult(HomePage.java:492)
at com.pwc.productcentral.SanityTestCases.WEB_68(SanityTestCases.java:812)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_68 -->
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_69" time="70.472"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_7" time="1.132"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_70" time="20.493"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_71" time="28.368"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_72" time="20.081"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_73" time="22.004"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_74" time="38.558"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_75" time="39.833"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_76" time="48.771"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_77" time="27.212"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_78" time="11.685"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_79" time="22.1"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_8" time="7.509"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_80" time="29.338"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_81" time="8.661"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_82" time="109.709"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_83" time="28.604"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_84" time="33.521"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_85" time="54.444"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_86" time="22.582"/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_9" time="15.652"/>
</testsuite> <!-- Default test -->
